odkx-src/sync-endpoint.rst

```diff
@@ -132,3 +143,129 @@ Assigning users to groups
 
     a. Navigate to the :guilabel:`memberUid` section.
     b. Click modify group members to manage members.
+    
+.. _sync-endpoint-ldap-utils:
+
+Using LDAP utils
+"""""""""""""""""""""""""
+
+  The ldap-service container has ldap-utils installed. If you'd prefer, you may use that tool set to administer the LDAP directory as well. Use this command to
+  access them,
+ 
+    .. code-block:: console
+  
+     $ docker exec $(docker ps -f "label=com.docker.swarm.service.name=${STACK_NAME}_sync" --format '{{.ID}}') <LDAPTOOL> <ARGS>
+     
+    The docker exec takes a container name and a command . The command is executed in the specified container . 
+    
+.. _sync-endpoint-advanced:
+
+Advanced
+--------
+
+.. _sync-endpoint-ldap-defaults:
+
+Editing the defaults of LDAP directory
+"""""""""""""""""""""""""""""""""""""""""""""
+    Modify the ldap.env file to configure the environment variables . The default settings are as follows 
+    
+      .. code-block:: console
+      
+       # openldap
+       LDAP_ORGANISATION=Open Data Kit            // name of your organisation
+       LDAP_DOMAIN=example.org                    // domain of your organisation
+       LDAP_READONLY_USER=true                    // enable the read only user
+       LDAP_READONLY_USER_PASSWORD=readonly       // password for read only user
+       LDAP_ADMIN_PASSWORD=admin                  // default password for admin account 
+
+       # phpldapadmin
+       PHPLDAPADMIN_LDAP_HOSTS=ldap-service   // This is for the phpLDAPadmin . In docker swarm this is the name of the service running LDAP . This can be eddited
+                                                 in the docker-compose.yml file
+                                                 
+   
+.. _sync-endpoint-custom-ldap:
+
+Using a Different LDAP directory
+""""""""""""""""""""""""""""""""""""""""""""""
+    
+    1. Create a folder in the sync-endpoint-default-setup directory and create a docker file inside it .
+    
+    2. Copy the bootstrap.ldif file from the OpenLDAP directory to the new directory .In the docker file Add the image of the ldap directory to be used and add 
+       the "COPY" command to copy the bootstrap.ldif file to the right path in the container .
+    
+    3. Run the following command to build the docker image :
+    
+     .. code-block:: console
+  
+       $ docker build -t odk/[LDAP_DIRECTORY_NAME] [ Folder conatining the docker file ]
+       
+    4. Replace the ldap-service image from docker-compose.yml with odk/[LDAP_DIRECTORY_NAME].
+    
+    5. Modify config/sync-endpoint/security.properties to fill in the Settings for LDAP server.
+       Set security.server.ldapUrl in security.properties to the new server url . The name of the service in swarm would be same ( ldap-service ) . So just change 
```

Comment:
This is not accurate. The preferences configured in `security.properties` (these lines: https://github.com/odk-x/sync-endpoint-default-setup/blob/0d97f322b6f3c48f86cdb47c9c1c7acf3f7bfb93/config/sync-endpoint/security.properties#L38-L51) all need to match the target LDAP server.